pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -55,12 +55,9 @@ stdenv.mkDerivation rec {
   pname = "inkscape";
   version = "1.2";
 
-  src = fetchFromGitLab {
-    owner = "inkscape";
-    repo = "inkscape";
-    rev = "dc2aedaf03f19bd890529a9e4720a02465fa1b6c";
-    sha256 = "sha256-h10E33THDM2Ci4uQ+7/KEmmnm6orAeV8seKa3IYPCGc=";
-    fetchSubmodules = true;
+  src = fetchzip {
```

Comment:
`src = fetchurl {…}` is much more common than `src = fetchzip {…}`. The manual [explains](https://nixos.org/manual/nixpkgs/stable/#fetchurl) the difference but not which to prefer. It makes sense to me that storing the source compressed (`fetchurl`) generally makes more sense than uncompressed (`fetchzip`)?